cardano-cli/cardano-cli.cabal

```diff
@@ -174,15 +174,23 @@ test-suite cardano-cli-pioneers
   type:                 exitcode-stdio-1.0
 
   build-depends:        base
+                      , aeson
                       , bech32
                       , cardano-api
                       , cardano-cli
                       , cardano-prelude
+                      , containers
                       , directory
                       , hedgehog
+                      , lens
```

Comment:
We don't use `lens` because we believe its not necessary to drag in all those additional dependencies for the sake of convenience. I think where you have used the `lens` accessors, you could have parsed the JSON and used the data type accessors instead.